srcpkgs/interception-tools/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'interception-tools'
+pkgname=interception-tools
+version=0.6.6
+revision=1
+wrksrc="tools-v${version}"
+build_style=cmake
+makedepends="boost-devel eudev-libudev-devel libevdev-devel yaml-cpp-devel"
+depends="glibc eudev-libudev libevdev yaml-cpp"
```

Comment:
All of the depends here should be detected automatically. Listing glibc also breaks musl builds for no good reason.

Reply:
I'll try to build without listing those.

Reply:
Cool, it worked! Is this because `-devel` versions of those are listed as `makedepends`? Can it also automatically detect `makedepends`?

Reply:
No. xbps-src will scan all ELF files in the package and figure out the shared libraries they need, then consult `common/shlibs` to see what packages provide said shared libraries.

`makedepends` are still the responsibility of the packager.